bom/runtime/pom.xml

```diff
@@ -34,6 +34,20 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Qpid JMS -->
+            <dependency>
+                <groupId>org.amqphub.quarkus</groupId>
+                <artifactId>quarkus-qpid-jms</artifactId>
+                <version>${quarkus-qpid-jms.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.qpid</groupId>
+                <artifactId>qpid-jms-client</artifactId>
+                <version>${qpid-jms.version}</version>
```

Comment:
I don't understand why this isn't in your BOM?

Reply:
As it isnt a bom, and I had only partially made the feedback change I was meant to, forgetting to remove the import. I have pushed it again.

From earlier comments when I was importing quarkus-qpid-jms-parent:
> The extension doesn't have boms, the referenced https://github.com/amqphub/quarkus-qpid-jms/blob/0.11.2/pom.xml#L91-L120 is where the versions are managed currently.
> 

For extra context, I based what quarkus-qpid-jms-parent does on what the extensions in Quarkus were doing when it was written and appear to still do, where they inherit quarkus-build-parent at their own parent pom and dont have any dep management of their own in runtime or deployment poms, so I guess they are getting the same dep management in both cases.

Reply:
User applications will import only the runtime part though.